tests/integration/suite/webhooks-ledger-committed-transaction-worker.go

```diff
@@ -50,10 +53,13 @@ var _ = WithModules([]*Module{modules.Ledger, modules.Webhooks}, func() {
 		)
 		Expect(err).ToNot(HaveOccurred())
 		Expect(response.StatusCode).To(Equal(http.StatusOK))
+		
 	})
 
 	When("creating a transaction", func() {
+		
 		BeforeEach(func() {
+			time.Sleep(1*time.Second)
```

Comment:
We should avoid this in tests. Tests should not depend on time. 

E.G You could implement a Ticker with manual declencher

Reply:
This is orchestration tests, I need let the caches of the worker and collector runner refresh.
Whether with a sleep() or a ticker, I have to wait at least one second to be able to trigger the rest.
(And ticker is wrapper for timer by the way, with a sleep() inside)

So I think I do not understand how to do it without wait.